packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -382,13 +402,29 @@ class FlutterErrorDetails with Diagnosticable {
   /// the console.
   final String library;
 
-  /// A human-readable description of where the error was caught (as opposed to
-  /// where it was thrown).
+  /// A [DiagnosticsNode] that provides a human-readable description of where
+  /// the error was caught (as opposed to where it was thrown).
+  ///
+  /// The node, e.g. an [ErrorDescription], should be in a form that will make
+  /// sense in English when following the word "thrown", as in "thrown while
+  /// obtaining the image from the network" (for the context "while obtaining
+  /// the image from the network").
+  ///
+  /// ```dart
+  /// ErrorDescription('while running async test code');
```

Comment:
Could this example be slightly more informative by showing how this is used within, e.g. `FlutterError.reportError`?

Reply:
Sure! Should I just use the same sample as above?

Reply:
Similar to that one, maybe one hat actually uses an ErrorDescription.